changes/ocd-3894.sql

```diff
@@ -0,0 +1,35 @@
+DROP TABLE IF EXISTS openchpl.change_request_website;
+DROP TABLE IF EXISTS openchpl.change_request_developer_details;
+
+CREATE TABLE IF NOT EXISTS openchpl.change_request_developer_demographic (
```

Comment:
Should this be plural?

```suggestion
CREATE TABLE IF NOT EXISTS openchpl.change_request_developer_demographics (
```